<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="revertButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>291</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>вернуть что до бинаризации было</string>
   </property>
  </widget>
  <widget class="QLabel" name="persontVolos">
   <property name="geometry">
    <rect>
     <x>807</x>
     <y>71</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadPhotoButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>загрузить фото</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>колличество волос</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="inverseCheckBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>инверсная бинаризация</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>%покрытия волосами</string>
   </property>
  </widget>
  <widget class="QLabel" name="countvolosLabel">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>100</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>порог</string>
   </property>
  </widget>
  <widget class="QPushButton" name="culculateCountVolos">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>посчитать волосы</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="Scenezoom" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>убрать лишние помехи</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>скилетизировать</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sizeCheckBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>реальный масштаб</string>
   </property>
  </widget>
  <widget class="Scenezoom" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>371</width>
     <height>331</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="toBinaryButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>бинаризовать</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scenezoom</class>
   <extends>QGraphicsView</extends>
   <header location="global">scenezoom.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
